### PR TITLE
feat: add google analytics to our documentation

### PR DESCRIPTION
**Pull request by Lenderson**
#### What is the purpose of this pull request?
Add google analytics to our documentation

#### What problem is this solving?
We are not tracking access to our documentation

#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Quality improvement (tests or refactors)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Trivial change (small fix or feature that doesn't impact functionalities)
- [ ] Requires change to documentation, which has been updated accordingly

#### How does this PR make you feel? [:link:](http://giphy.com/)
data-analysis

### CODE REVIEW

File: packages/admin-ui-docs/docusaurus.config.js [hunk 1]
```diff
@@ -30,6 +30,10 @@ const config = {
         theme: {
           customCss: require.resolve('./src/css/custom.scss'),
         },
+        gtag: {
+          trackingID: 'G-K6LT0G8PH7',
```
Comment (lucasaarcoverde) [inline packages/admin-ui-docs/docusaurus.config.js:34]: Isn't this sensitive content?

Comment (Lenderson) [reply packages/admin-ui-docs/docusaurus.config.js:34]: This is just the id to where we are sending our tracking data and this id will also appears in the request when the data is sent to GA.

Comment (kevinch) [reply packages/admin-ui-docs/docusaurus.config.js:34]: It's a "public" ID available in all source codes on html pages, no problem.

File: packages/admin-ui-docs/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@vtex/admin-ui-docs",
-  "version": "0.4.5",
+  "version": "0.4.3",
```
Comment (kevinch) [inline packages/admin-ui-docs/package.json:3]: 5 to 3?

**Review by kevinch — APPROVED** [review]

**Pull request merged**